webapp/components/interop-summary.js

```diff
@@ -197,9 +197,24 @@ class InteropSummary extends PolymerElement {
     return scores !== null && scores !== undefined;
   }
 
+  // roundScore defines the rounding rules for the top-level scores.
+  roundScore(score) {
+    // Round down when a score is above the 975 breakpoint.
```

Comment:
The option we settled on in the interop team is to special-case 100 only. So we would round the score, and if it's 100 but `score` is not 1000, we change it to 99.

Reply:
Resolved!

Reply:
Thanks Kyle!